sections/builder.js

```diff
@@ -715,12 +715,12 @@ export default [
     type: "method",
     method: "crossJoin",
     example: ".crossJoin(column, ~mixed~)",
-    description: "",
+    description: "Cross join conditions are only supported in MySQL and sqlite3. For join conditions rather use innerJoin.",
     children: [
       {
         type: "runnable",
         content: `
-          knex.select('*').from('users').crossJoin('accounts', 'users.id', 'accounts.user_id')
+          knex.select('*').from('users').crossJoin('accounts')
```

Comment:
Could this be an additional example to start docs and the replaced example would be the second one? Now this PR is removing one example, which would be still good to have in docs.